app/src/androidTest/java/org/standardnotes/notes/TestHelper.java

```diff
@@ -0,0 +1,41 @@
+package org.standardnotes.notes;
+
+import android.app.Activity;
+import android.content.res.Configuration;
+import android.util.DisplayMetrics;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewParent;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+public class TestHelper {
+
+    public static Matcher<View> childAtPosition(
+            final Matcher<View> parentMatcher, final int position) {
+
+        return new TypeSafeMatcher<View>() {
+            @Override
+            public void describeTo(Description description) {
+                description.appendText("Child at position " + position + " in parent ");
+                parentMatcher.describeTo(description);
+            }
+
+            @Override
+            public boolean matchesSafely(View view) {
+                ViewParent parent = view.getParent();
+                return parent instanceof ViewGroup && parentMatcher.matches(parent)
+                        && view.equals(((ViewGroup) parent).getChildAt(position));
+            }
+        };
+    }
+
+    public static boolean isScreenSw600dpAndLandscape(Activity activity) {
```

Comment:
So I don't like that the normal code references a TestHelper - this probably wants moving into a separate class. The other problem is that this duplicates the resource sw600-land config, which is fine for tests but I'm lairy about using it in production. If that is changed and not this, we have a bug in the future, or if this doesn't accurately reflect what the resource config does, and I'm not 100% sure it will.